Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM dmstr/php-yii2:7.0-fpm-1.9-beta2-alpine-nginx
```

Comment:
Actually, the one in the root folder could become kind of an official image, I am working with @mikehaertl on something similar.

Or should this be moved to tests also?